controllers/gatekeeper_controller.go

```diff
@@ -225,7 +247,7 @@ func (r *GatekeeperReconciler) SetupWithManager(mgr ctrl.Manager) error {
 				return oldGeneration != newGeneration
 			},
 			DeleteFunc: func(e event.DeleteEvent) bool {
```

Comment:
You can just delete `DeleteFunc` since if it's not set, it defaults to reconciling.